react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/RNCustomerCenterModule.kt

```diff
@@ -84,9 +90,63 @@ internal class RNCustomerCenterModule(
     private fun presentCustomerCenterFromActivity(
         activity: Activity
     ) {
+        val customerCenterListener = createCustomerCenterListener()
+        Purchases.sharedInstance.customerCenterListener = customerCenterListener
         val intent = ShowCustomerCenter()
             .createIntent(activity, Unit)
         activity.startActivityForResult(intent, REQUEST_CODE_CUSTOMER_CENTER)
     }
 
+    private fun createCustomerCenterListener(): CustomerCenterListener {
+        return object : CustomerCenterListenerWrapper() {
+            override fun onFeedbackSurveyCompletedWrapper(feedbackSurveyOptionId: String) {
+                val params = WritableNativeMap().apply {
+                    putString("feedbackSurveyOptionId", feedbackSurveyOptionId)
+                }
+                sendEvent("onFeedbackSurveyCompleted", params)
+            }
+
+            override fun onManagementOptionSelectedWrapper(action: String, url: String?) {
+                val params = WritableNativeMap().apply {
+                    putString("option", action)
+                    putString("url", url)
+                }
+                sendEvent("onManagementOptionSelected", params)
+            }
+
+            override fun onShowingManageSubscriptionsWrapper() {
+                sendEvent("onShowingManageSubscriptions", null)
+            }
+
+            override fun onRestoreCompletedWrapper(customerInfo: Map<String, Any?>) {
+                val params = WritableNativeMap().apply {
+                    putMap("customerInfo", RNPurchasesConverters.convertMapToWriteableMap(customerInfo))
+                }
+                sendEvent("onRestoreCompleted", params)
+            }
+
+            override fun onRestoreFailedWrapper(error: Map<String, Any?>) {
+                val params = WritableNativeMap().apply {
+                    putMap("error", RNPurchasesConverters.convertMapToWriteableMap(error))
+                }
+                sendEvent("onRestoreFailed", params)
+            }
+
+            override fun onRestoreStartedWrapper() {
+                sendEvent("onRestoreStarted", null)
+            }
+        }
+    }
+
+    private fun sendEvent(eventName: String, params: WritableMap?) {
+        reactContext.runOnUiQueueThread {
+            try {
+                reactContext
+                    .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter::class.java)
+                    .emit(eventName, params)
+            } catch (e: Exception) {
+                Log.e(NAME, "Error sending event $eventName", e)
+            }
+        }
+    }
```

Comment:
What's the difference between this events, and [this](https://github.com/RevenueCat/react-native-purchases/pull/1209/files#diff-68a995f0356fff3a1453ac899055f7e25c4c4a6db57e0deb9bbd137a2b8e5b10R67) ??